msautotest/mssql/cluster_mssql.map

```diff
@@ -0,0 +1,51 @@
+#
+# Test a clustered layer with the MSSQL driver plugin
```

Comment:
is the issue specific to using mssql ? Can't the be reproduced with shapefiles ?

Reply:
The issue is around a NULL `layerinfo->conn` in the database driver when trying to get the extent from the database. Shapefiles would be unaffected, but I'd image the PostGIS and Oracle drivers would have the same problem.